Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -74,6 +76,7 @@ - (BOOL)installWithOptions:(nonnull SentryOptions *)options
             return event;
         }];
 
+    [SentryDependencyContainer.sharedInstance.reachability addObserver:self];
```

Comment:
`m`: I know we already use it like this in the `SentryHttpTransport`, but I saw cases in the test logs of CI showing that the simulator became online or offline during tests, triggering the `connectivityChanged` callback causing tests to flake. I would prefer if we could find a way to use test implementation per default for all tests in the `SentrySessionReplayIntegrationTests` to avoid that.

Reply:
Good idea